target/mips/cp0_helper.c

```diff
@@ -1766,6 +1791,7 @@ target_ulong helper_dvpe(CPUMIPSState *env)
 
 target_ulong helper_evpe(CPUMIPSState *env)
 {
+    qemu_mutex_lock_iothread();
```

Comment:
This causes an assertion failure: `mips_vpe_wake` as of 256eb7ee587 also grabs the iothread lock.

Reply:
Fixed